### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by juanjgit**
Extra backslash removed

### CODE REVIEW

**Comment by juanjgit** [comment]
CLA Signed

File: pixiecore/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ alpha release of CoreOS, with automatic login:
 
 ```shell
 sudo pixiecore boot \
-  https://alpha.release.core-os.net/amd64-usr/current/coreos_production_pxe.vmlinuz \
```
Comment (danderson) [inline pixiecore/README.md]: This one is still required, since this is a single shell command.

**Comment by panchoh** [comment]
Closed via #23.

**Comment by danderson** [comment]
Closing since panchoh++ sent a corrected pull request.

**Pull request closed without merging**